apis_core/apis_entities/edit_generic.py

```diff
@@ -277,7 +277,7 @@ class GenericEntitiesDeleteView(DeleteView):
     #     app_label='apis_entities', model='tempentityclass').model_class()
     model = importlib.import_module("apis_core.apis_entities.models").TempEntityClass
     template_name = getattr(
-        settings, "APIS_DELETE_VIEW_TEMPLATE", "apis_entities/confirm_delete.html"
```

Comment:
is that correct? was the confirm delete also moved to root?

Reply:
Yes, there was already one `confirm_delete.html` template in `apis_entities` and the one shipped with `webpage`. They were identical, I moved one of them to the root template folder of `apis_metainfo` and deleted the one in `apis_entities`